azure_functions_worker/testutils_lc.py

```diff
@@ -129,6 +135,12 @@ def _find_latest_mesh_image(cls,
         cls._mesh_images[host_major] = image_tag
         return image_tag
 
+    @staticmethod
```

Comment:
I don't see the code where you filter our *-upgrade images. Could you please check and bring in that?